scripts/update_plugins.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+PLUGIN_SERVER_IMAGE="$(/usr/local/bin/docker-compose ps -q plugin_server)"
+export PLUGIN_SERVER_IMAGE
+# echo "Removing existing plugin folder in Docker container.."
+# docker-compose exec plugin_server rm -rf /app/plugins/
+echo "Copying current plugin folder to Docker container.."
+docker cp ./server/plugins/ "${PLUGIN_SERVER_IMAGE}":/app/
+echo "Done."
```

Comment:
Can't the dockerfile just do the copy?

Reply:
It does copy on start. It is just for extra security later on, I will explain down below.